scss/base/_colors.scss

```diff
@@ -135,6 +150,31 @@ $purple-dark: #582e63;
 .bg-purple-dark { background-color: $purple-dark; }
 .bc-purple-dark { border-color: $purple-dark; }
 
+$ds-grey-10:  #f9f9f9;
```

Comment:
I know these are redundant (this is the same as `grey-3`) but I'm worried I'll miss something when I make sweeping changes across the app. Keeping both to play it safe

Reply:
Good thinking. I agree that we keep both as well